resources/saas-boost-svc-onboarding.yaml

```diff
@@ -309,6 +309,11 @@ Resources:
               - ecr:DeleteRepository
             Resource:
               - !Sub arn:${AWS::Partition}:ecr:${AWS::Region}:${AWS::AccountId}:repository/*
+          - Effect: Allow
+            Action:
+              - cloudfront:GetDistribution
+            Resource:
+              - !Sub arn:${AWS::Partition}:cloudfront::${AWS::AccountId}:distribution/*
```

Comment:
What do we need this for? I assume this is a preparatory change for the upcoming Keycloak integration PR?